### ISSUE TITLE
Don't tech gate dish antennas

### ISSUE THREAD

**Issue by Capkirk123** [KSP-RO/RP-1#2012]
The current placement of dish antennas in the tech tree is still mostly a holdover from RT, where each antenna had a fixed comms range. Now that RA is the only(?) officially supported comms mod, I believe all fixed dishes, and potentially some folding ones, should be moved much earlier in the tech tree.

**Comment by byt3c** [comment]
Agreed. I know historically those were put on some specific spaceships, but it prevents players from creativity and building around certain limitations using other sources. We should consider if strict historical limits on actual parts are not forcing players to simply repeat space race instead of procedurally give them environment to do so in a way they want it.
